deployment-platforms/aws-lambda/package.json

```diff
@@ -12,5 +12,5 @@
     "@prisma/cli": "2.2.0",
     "serverless-dotenv-plugin": "2.4.2"
   },
-  "license": "MIT"
+  "license": "Apache 2.0"
```

Comment:
If we are doing this, it should be done in a separate PR please that can be merged on its own.